docs/source/solidity_developers_guide/seth_rpc.rst

```diff
@@ -0,0 +1,93 @@
+********
```

Comment:
@chenette @agunde406 while I don't disagree with any of Anne's requested changes, due to time constraints we should just focus on fixing the small pieces now inconsistencies, grammar, typos. Since these comments will stay on the PR even after it is merged, they can be addressed in a future PR that focuses on making the documents less "design-y". Anne, could you make a task in JIRA along these lines?

Reply:
Yes, that sounds good to me.  I'll create a JIRA task to rewrite the design-y docs (probably next week).